cli/src/cli_util.rs

```diff
@@ -1994,6 +2012,24 @@ pub fn print_trackable_remote_branches(ui: &Ui, view: &View) -> io::Result<()> {
     Ok(())
 }
 
+pub fn check_dropped_signature(
+    ui: &Ui,
+    old_commit: &Commit,
+    rewritten: &Commit,
+) -> Result<(), std::io::Error> {
+    if old_commit.is_signed() && !rewritten.is_signed() {
+        writeln!(ui.warning(), "Dropped a commit signature")?;
+    }
+    Ok(())
+}
+
+pub fn print_dropped_signatures(ui: &Ui, num_dropped: usize) -> Result<(), std::io::Error> {
+    if num_dropped > 0 {
+        writeln!(ui.warning(), "Dropped {} commit signatures", num_dropped)?;
+    }
+    Ok(())
+}
```

Comment:
might be better to extract a helper that takes `RebaseCounts` and print both "Rebased N descendant commits" message and "Dropped M commit signatures".

Not all callers can't be migrated to it, but there are many?

Reply:
Yea makes sense - this is now done.